security-framework/src/os/macos/certificate.rs

```diff
@@ -83,6 +88,15 @@ impl SecCertificateExt for SecCertificate {
             }
         }
     }
+
+    /// Returns the SHA-256 fingerprint of the certificate.
+    fn fingerprint(&self) -> Result<String, CFError> {
+        let data = CFData::from_buffer("The quick brown fox jumps over the lazy dog".as_bytes());
+        let hash = Builder::new()
+            .type_(DigestType::md5())
```

Comment:
md5?

Reply:
Forgot to stage this file to the final commit. 😅 Thanks.